### PR TITLE
Add dotdigital

### PR DESCRIPTION
**Pull request by jamesmacwhite**
Add dotdigital a multi channel marketing tool.

Currently only supports SMS 2FA.

### CODE REVIEW

File: entries/d/dotdigital.com.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "dotdigital": {
```
Comment (Carlgo11) [inline entries/d/dotdigital.com.json]:
```suggestion
  "Dotdigital": {

```

File: entries/d/dotdigital.com.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+  "dotdigital": {
+    "domain": "dotdigital.com",
+    "tfa": [
+      "sms"
+    ],
+    "documentation": "https://support.dotdigital.com/hc/en-gb/articles/212212678-Secure-your-account-with-two-factor-authentication",
```
Comment (Carlgo11) [inline entries/d/dotdigital.com.json]: For Zendesk URLs the part after the "-" is redundant and only causes issues when someone decides to rename the page.
```suggestion
    "documentation": "https://support.dotdigital.com/hc/en-gb/articles/212212678",

```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

File: entries/d/dotdigital.com.json (hunk 1 above)
Comment (jamesmacwhite) [reply entries/d/dotdigital.com.json]: I understand if it is on grammar grounds but the name of the product is all lower case and written as dotdigital in materials. Do you still want me to capitalise it?

File: entries/d/dotdigital.com.json (hunk 2 above)
Comment (jamesmacwhite) [reply entries/d/dotdigital.com.json]: That's good to know on ZendDesk URLs. I'll make a note of that. I'll amend this one.

**Review by Carlgo11 — APPROVED** [review]

**Pull request merged**